### DISCUSSION TITLE
Worker_1 and requeuer_1 stops working immediately after I run postal start.

### DISCUSSION THREAD

**Discussion by NaumanAnsari30** [postalserver/postal#2146] [Help with using Postal]
Hi Postal Community,

I am having this issue since last 3 days. Whenever I send message from postal server it gets pending and don't redeliver.

I am assuming I made some mistakes when installing RabittMQ database. I changed password and vhost settings that's making trouble.

If anyone can tell me is it possible that I can change these settings again by any means or I have to install the postal again from start.

Thanks in advance.

![Screenshot from 2022-09-10 11-07-11](https://user-images.githubusercontent.com/70816841/189471945-e0c550f8-c5ef-4a79-83b2-2c1d02f2c9d9.png)
![Screenshot from 2022-09-10 11-06-24](https://user-images.githubusercontent.com/70816841/189471950-bc34cffc-cb90-4876-9907-9c874ecc032a.png)
![Screenshot from 2022-09-10 11-02-35](https://user-images.githubusercontent.com/70816841/189471959-f70c028d-3ba1-4d01-adfc-1b293a589bfd.png)

**Comment by willpower232** [comment]
Do you mean you changed the password for your RabbitMQ but need to update the Postal config file? Or are the ones you think you have not working?

Either way, you shouldn't need to reinstall Postal to resolve this, in the absolute worst case scenario you need to re-install your RabbitMQ unless you have some admin access to reset the password in there.

If your RabbitMQ is in docker, this might be as easy as deleting any persistent volume you have provided it.

**Reply by NaumanAnsari30** [reply]
Hi Will,

I am a beginner with all these things I just installed the Postal by copy pasting the commands from some blog. I have reinstalled everything from scratch but still same problem exists.

I don't know where am I making mistake...!

If you can guide me in simple steps then I would be grateful.

Thanks.

**Reply by willpower232** [reply]
How does this blog talk about installing RabbitMQ? You should have a look at the relatively new docs website https://docs.postalserver.io/install/prerequisites
